test/draw/gpu/ToRasterTest.ooc

```diff
@@ -40,15 +40,31 @@ ToRasterTest: class extends Fixture {
 		gpuImage free()
 		raster free()
 	}
+	toRasterAsync: func (sourceImage: RasterImage) {
+		gpuImage := context createImage(sourceImage)
+		future := gpuImage toRasterAsync() as _ToRasterFuture
+		future wait()
+		raster := future getResult(null)
+		expect(raster != null)
+		distance := raster distance(sourceImage)
```

Comment:
You assign `distance` here but never use it on the next line. That's why PRInt fails.